recipe/meta.yaml

```
@@ -85,43 +88,31 @@ outputs:
         - cmake
         - ninja-base
         # pkg-config on win pulls in python
-        # adding here explicitly to pull in the same version as in host
-        - python                         # [win]
-        - {{ cdt('libselinux') }}        # [linux]
-        - {{ cdt('libxau-devel') }}      # [linux]
-        - {{ cdt('libxcb') }}            # [linux]
-        - {{ cdt('libxdamage-devel') }}  # [linux]
-        - {{ cdt('libxext-devel') }}     # [linux]
-        - {{ cdt('libxfixes-devel') }}   # [linux]
-        - {{ cdt('libxi-devel') }}       # [linux]
-        - {{ cdt('libxrender-devel') }}  # [linux]
-        - {{ cdt('libxxf86vm') }}        # [linux]
-        - {{ cdt('mesa-libegl-devel') }} # [linux]
-        - {{ cdt('mesa-libgbm') }}       # [linux]
-        - {{ cdt('mesa-libgl-devel') }}  # [linux]
-        - {{ cdt('mesa-dri-drivers') }}  # [linux]
+        - python                        # [win]
       host:
         - python
-        - numpy {{ numpy }}
         - eigen 3.4.0
-        - ffmpeg 6                  # [not win]
+        - ffmpeg {{ ffmpeg }}           # [not win]
         - freetype {{ freetype }}
+        - gst-plugins-base {{ gst_plugins_base }}
+        - gstreamer {{ gstreamer }}
         # harfbuzz, glib, gettext are both needed for freetype support
         - harfbuzz {{ harfbuzz }}
-        - libglib {{ glib }}
-        - gstreamer 1.24
-        - gst-plugins-base 1.24
         - hdf5 {{ hdf5 }}
         - jpeg {{ jpeg }}
-        - openjpeg 2.5.2
-        - libiconv 1.16             # [unix]
-        - libpng  {{ libpng }}
-        - libprotobuf 5.29.3
-        - libabseil 20250127.0
+        - libabseil {{ libabseil }}
+        - glib {{ glib }}
```

```suggestion
        - glib {{ glib }}
        - libglib {{ glib }}
```
I believe this will remove a linter warning regarding run dep not in host.

---

Could also just remove it from run since it will be put there with the `run_export` but that's true for most of the things in this recipe.